src/Synapse/Synapse/help/Az.Synapse.md

```diff
@@ -77,6 +77,15 @@ This feature is in a limited preview, initially accessible only to certain subsc
 ### [Get-AzSynapseSqlPool](Get-AzSynapseSqlPool.md)
 Gets a Synapse Analytics SQL pool.
 
+### [Get-AzSynapseSqlPoolVulnerabilityAssessmentScanRecord](Get-AzSynapseSqlPoolVulnerabilityAssessmentScanRecord.md)
+Gets a Synapse Analytics SQL pool vulnerability assessment scan record.
+
+### [Convert-AzSynapseVulnerabilityAssessmentScan](Convert-AzSynapseVulnerabilityAssessmentScan.md)
+Converts to a Synapse Analytics SQL pool vulnerability assessment scan record.
+
+### [Start-AzSynapseVulnerabilityAssessmentScanInitialziation](Start-AzSynapseVulnerabilityAssessmentScanInitialziation.md)
+Start a Synapse Analytics SQL pool vulnerability assessment scan record.
```

Comment:
Do you modify this file manually? Usually we use the `Update-MarkdownHelpModule` command

Reply:
Got it, thanks. Yeah, I modified it manually

Reply:
Please use `Update-MarkdownHelpModule` to  update the entries again. These entries should be in alphabetical order.

---
In reply to: [542156034](https://github.com/Azure/azure-powershell/pull/13690#discussion_r542156034) [](ancestors = 542156034)